gammapy/utils/pbar.py

```diff
@@ -0,0 +1,36 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Utilities for progress bar display"""
+import logging
+
+log = logging.getLogger(__name__)
+
+try:
+    from tqdm.auto import tqdm
+except ImportError:
+    class tqdm():
+        def __init__(self, iterable, disable=True, **kwargs):
+            self.disable = disable
+            self._iterable = iterable
+            if not self.disable:
+                log.info(
+                    f"Tqdm is currently not installed. Visit https://tqdm.github.io/"
+                )
+
+        def update(self, x):
+            pass
+
+        def __iter__(self):
+            return self._iterable.__iter__()
+
+        def __next__(self):
+            return self._iterable.__next__()
+
+global SHOW_PROGRESS_BAR
```

Comment:
Last comment: this shouldn't be needed. Python automatically looks up a variable in the outer scope, if it is not part of the inner scope (the "progress_bar" method here)...

Reply:
@adonath yes indeed. I removed it